scripts/detectNotUsingGitPush1by1.fsx

```diff
@@ -814,16 +814,32 @@ let notUsingGitPush1by1 =
 
         let json = GitHubApiCall url
 
-        json.Contains "\"check_suites\":[]"
+        not(json.Contains "\"check_suites\":[]")
     )
-    |> Seq.contains true
 
-if notUsingGitPush1by1 then
+let notActivatedGitHubActions = not(Seq.contains true hasCiStatus)
+
+if notActivatedGitHubActions then
```

Comment:
@realmarv so that here you can change this to `if not gitHubActionsEnabled then`